_benchmark/installing-benchmark.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: default
+title: Installing OpenSearch Benchmark
+nav_order: 5
+has_children: false
+---
+
+# Installing OpenSearch Benchmark
+
+You can install OpenSearch Benchmark directly on a host running Linux or macOS or you can run OpenSearch Benchmark in a Docker container on any compatible host. This page describes high-level considerations for your OpenSearch Benchmark host, as well as instructions for installing OpenSearch Benchmark.
+
+## Docker limitations
```

Comment:
Do we want to lead with this as a heading? Can we make it a note instead?

Reply:
Or maybe add it lower down where you are discussing the Docker install?

Reply:
I'll lower it down.